src/main/java/com/iexec/worker/config/WorkerConfigurationService.java

```diff
@@ -56,9 +56,9 @@
     @Getter
     private boolean developerLoggerEnabled;
 
-    @Value("${worker.tee-compute-max-heap-size-GB}")
+    @Value("${worker.tee-compute-max-heap-size-gb}")
     @Getter
-    private int teeComputeMaxHeapSizeGB;
+    private int teeComputeMaxHeapSizeGb;
```

Comment:
1 Byte = 8 bits (but please discard my comment here..!)

Reply:
It seems obvious to me that it is in Gega **Bytes**, but I will use the "B" just to remove any possible confusion.

Reply:
Still don't know myself if I was serious our not :thinking: